egs/librispeech/s5/local/chain/run_tdnn_6z.sh

```diff
@@ -0,0 +1,208 @@
+set -e
+
+# configs for 'chain'
+affix=
+stage=11
+train_stage=-10
+speed_perturb=true
+get_egs_stage=-10
+dir=exp/chain/tdnn_6z  # Note: _sp will get added to this if $speed_perturb == true.
+decode_iter=
+
+# TDNN options
+# this script uses the new tdnn config generator so it needs a final 0 to reflect that the final layer input has no splicing
+# smoothing options
+pool_window=
+pool_type='none'
+pool_lpfilter_width=
+self_repair_scale=0.00001
+# training options
+num_epochs=4
```

Comment:
These variables are a bit redundant, you could reduce the size of your script by removing any script which is not used more than once.